test/routes/testSoldierRoutes.js

```diff
@@ -0,0 +1,98 @@
+const expect = require('chai').expect;
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+const testServer = require("../../start.js").server;
+const MongoClient = require('mongodb').MongoClient;
+
+const DBurl = 'mongodb://localhost:27017';
```

Comment:
`dbUrl`
I feel you, but programming standards overcome habits like writing DB in big letters